server/src/services/services.go

```diff
@@ -40,7 +40,9 @@ type Boards interface {
 	GetColumn(ctx context.Context, boardID, columnID uuid.UUID) (*dto.Column, error)
 	ListColumns(ctx context.Context, boardID uuid.UUID) ([]*dto.Column, error)
 
-	FullBoard(ctx context.Context, boardID uuid.UUID) (*dto.Board, []*dto.BoardSessionRequest, []*dto.BoardSession, []*dto.Column, []*dto.Note, []*dto.Reaction, []*dto.Voting, []*dto.Vote, error)
+  FullBoard(ctx context.Context, boardID uuid.UUID) (*dto.Board, []*dto.BoardSessionRequest, []*dto.BoardSession, []*dto.Column, []*dto.Note, []*dto.Reaction, []*dto.Voting, []*dto.Vote, error)
+	BoardOverview(ctx context.Context, boardID uuid.UUID) (*dto.Board, []*dto.BoardSession, []*dto.Column, error)
+	GetBoards(ctx context.Context, userID uuid.UUID) ([]*dto.Board, error)
```

Comment:
Consider returning only a list of board IDs with GetBoards instead of fetching the boards twice - once to get all boards of a user and a second time when you fetch the boards for the overview. This approach is common in API design and would improve the efficiency of the process.